mailer.js

```diff
@@ -134,22 +134,31 @@ module.exports = function (log) {
     // Build a first cut at a device description,
     // without using any new strings.
     // Future iterations can localize this better.
-    var parts = []
-    if (message.uaBrowser) {
-      var browser = message.uaBrowser
-      if (message.uaBrowserVersion) {
-        browser += ' ' + message.uaBrowserVersion
-      }
-      parts.push(browser)
+    var translator = this.translator(message.acceptLanguage)
+    var uaBrowser = message.uaBrowser
+    var uaOS = message.uaOS
+    var uaOSVersion = message.uaOSVersion
+    
+    if (uaBrowser && uaOS) {
+      return translator.format(translator.gettext('%(uaBrowser)s on %(uaOS)%(uaOSVersion)'), { uaBrowser: uaBrowser, uaOS: uaOS, uaOSVersion: uaOSVersion })
```

Comment:
hm interesting that there is no space between `%(uaOS)%(uaOSVersion)` but tests still have a space between OS and Version